website/docs/configuration/index.md

```diff
@@ -38,7 +38,43 @@ The priority applied names are:
 - `.markuplintrc.yml`
 - `.markuplintrc.js`
 - `.markuplintrc.cjs`
+- `.markuplintrc.ts`
 - `markuplint.config.js`
 - `markuplint.config.cjs`
+- `markuplint.config.ts`
 
 `.markuplintrc`'s format is JSON (with comment) and also YAML.
+
+#### JSON
+
+```json
+{
+  "extends": ["markuplint:recommended"]
+}
+```
+#### YAML
+
+```yaml
+extends:
+  - markuplint:recommended
+```
+
+#### JavaScript
+
+```js
+module.exports = {
+  extends: ['markuplint:recommended']
+};
+```
+
+#### TypeScript
+
+```ts
+import type { Config } from '@markuplint/ml-config';
+
+const config: Config = {
+  extends: ['markuplint:recommended']
+};
+
+export default config;
+```
```

Comment:
I have added simple examples of each format.

Reply:
Thank you for adding examples! 🎉 

Could you please add `class=config` to all the code blocks you have added this time?
This will apply the style for the configuration file.


example:
````
```json class=config
````

Reply:
@kagankan I did it in [ed184d7](https://github.com/markuplint/markuplint/pull/919/commits/ed184d7cea005b04e6fedab16bade0ed227efb37)

<img width="978" alt="スクリーンショット 2023-04-26 23 30 02" src="https://user-images.githubusercontent.com/20086673/234608882-14a82c6c-9e8d-42b8-9190-e3712cf602cf.png">

Reply:
Thanks!